neqo-transport/src/cc/classic_cc.rs

```diff
@@ -174,9 +189,19 @@ impl<T: WindowAdjustment> CongestionControl for ClassicCongestionControl<T> {
         acked_pkts: &[sent::Packet],
         rtt_est: &RttEstimate,
         now: Instant,
+        cc_stats: &mut CongestionControlStats,
     ) {
         let mut is_app_limited = true;
         let mut new_acked = 0;
+
+        // Supplying `rtt_est.pto(true)` here is best effort not to have to track
+        // `recovery::Loss::confirmed()` all the way down to the congestion controller. Having too
+        // big a PTO does no harm here and usually the connection should be confirmed at this point
+        // anyway.
```

Comment:
```suggestion
        // big a PTO does no harm here.
```

In an 1-RTT handshake, the client will send its initial, then receive the server's initial including an ACK from the server. That moves the connection into `State::Connected`. The ACK should call `on_packets_acked`. Only on the second round-trip does the client receive the `HANDSHAKE_DONE` frame from the client, subsequently going into `State::Confirmed`.

Long story short, I think the above is confusing. I suggest removing it.